### ISSUE TITLE
Reset Zenodo contributors, etc.

### ISSUE THREAD

**Issue by effigies** [poldracklab/fmriprep-rodents#5]
I would be inclined to drop most fMRIPrep contributors from the fmriprep-rodents list. I think it will be understood this is a derivative effort and that the people listed as contributors are involved in this specific project.

Thoughts?

**Comment by mgxd** [comment]
I think that's good - maybe we can add a little blurb in the readme (or somewhere else) linking to fMRIPrep contributors

**Comment by eilidhmacnicol** [comment]
I agree. It's also possible that contributors to fMRIPrep may oppose being directly associated with a project for preclinical research. For some researchers, preclinical research is an ethical line that they don't want to cross, even if they have nothing to do with data collection.

Citing fMRIPrep and fMRIPrep-rodents contributors separately would minimise those issues.

**Comment by eilidhmacnicol** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Zenodo](https://trello.com/c/tYkF7l2b/1-zenodo)
